### ISSUE TITLE
WebGL on Windows

### ISSUE THREAD

**Issue by kaamui** [qtwebkit/qtwebkit#910]
Hello @annulen 

I have some issues rendering WebGL content on Windows, where it works correctly on Ubuntu and Mac. Is there something to enable that is not enabled ba default on Windows, when compiling qtwebkit ?

As you can see on the right screen, in an internal navigator using qtwebkit, the molecules don't render..

Same issue on [WebGL water](http://madebyevan.com/webgl-water/). Any idea of what is causing this ?

![Capture d’écran de 2019-09-27 09-31-03](https://user-images.githubusercontent.com/11720903/65753121-99583f80-e10e-11e9-953a-6142e1175ce1.png)

**Comment by annulen** [comment]
cc @Emdek

**Comment by kaamui** [comment]
Same issue in Otter Browser

![Capture du 2019-09-27 12-36-29](https://user-images.githubusercontent.com/11720903/65763394-9156ca80-e123-11e9-91da-ef8da776ff43.png)

**Comment by Emdek** [comment]
Yep, it seems to not work for me either right now (MSVC and mingw), but AFAIR it used in the past, using the same example.

**Comment by annulen** [comment]
When did it stop working?

**Comment by kaamui** [comment]
It won't probably help you at all but I have it working with Qt5.5 (so with qtwebkit) except that there is an issue with the coordinates of the view in WebGL

Also, in #525 you are referring to this WebGL water page

**Comment by annulen** [comment]
Do you see anything interesting in stout/stderr or JS console?

**Comment by annulen** [comment]
If you are using official binaries with "dynamicgl" feature, make sure that ANGLE plugin can be loaded

**Comment by Emdek** [comment]
@annulen, it's hard to tell, I'm testing it rarely, but I think that it worked around alpha3 (these much older biinaries).

**Comment by kaamui** [comment]
> Do you see anything interesting in stout/stderr or JS console?

Nope sorry, I also have an issue with qDebug() not displaying anything even in debug mode, so I don't think I would see anything even if some js logs were sent. 

@Emdek As the same issue is found in your browser, you might have better chances to see if some logs could be found ?

**Comment by annulen** [comment]
@kaamui I've just tested it myself and it works fine.

How do you test it? In Otter it seems to be a packaging problem - ANGLE dlls were not included with application

**Comment by annulen** [comment]
You can try DebugView to see logs

**Comment by annulen** [comment]
Seems like ANGLE-based WebGL support is currently broken, only native OpenGL works
